docs/asyncapi.yaml

```diff
@@ -1,10 +1,10 @@
 asyncapi: 2.4.0
 info:
   title: FFC Payment Statement Generator
-  version: '1.0.0'
+  version: 1.0.1
   description: Generate PDF payment statements
 channels:
-  payment/request:
+  ffc-pay-statements/ffc-pay-statement-generator:
```

Comment:
Wouldn't include the Subscription name because as per our naming conventions, it should be the name of the service :)

Reply:
Or, whatcha think on this?

I think Confluence page says to use both (suppose to help us devs quickly see which Topic/Sub to send msg to).?

Reply:
Yeah the Confluence page says to use Topic Name/Subscription Name - I also think you suggested in one of your previous comments that the channel name should follow said naming convention on the Confluence page [https://eaflood.atlassian.net/wiki/spaces/SFI/pages/4177723458/AsyncAPI+Structure#Useful-Tips]

Reply:
Sounds good -- let's enforce it

Reply:
In that case I will leave as is :)